pages/integrations/github.md.erb

```diff
@@ -0,0 +1,58 @@
+# GitHub
+
+Buildkite can connect to your GitHub repository and use the Commit Status API to update the status of commits in Pull Requests.
+
+To complete this integration, you will need admin privileges for your GitHub repository.
+
+<%= toc %>
+
+## Connecting Buildkite and GitHub
+
+To connect your GitHub account open Buildkite’s Personal Settings, choose [Connected Apps](https://buildkite.com/user/connected-apps) then click the GitHub Connect button:
+
+<%= image "personal-settings.png", width: 1467/2, height: 565/2, alt: "Screenshot of the Buildkite Connected Apps screen" %>
+
+You'll be prompted to give permission for Buildkite to post status
+updates, and then redirected back to your Connected Apps page.
+
+## Setting up a new pipeline for a GitHub repository
+
+1. Create a new pipeline on Buildkite, using your GitHub repository URL in the form `git@github.com:your/repo`.
+
+    <%= image "new-pipeline.png", width: 1464/2, height: 744/2, alt: "Screenshot of adding a new pipeline " %>
+
+2. Make sure to add at least one command to your pipeline, before clicking next.
+
+3. Follow the instructions on the page to set up a webhook:
+
+    1. Add a new webhook in Github.
+    2. Paste in the the provided webhook URL.
+    3. Select which events trigger the webhook.
+
+## Branch configuration and Settings
+
+You can edit the GitHub settings for each pipeline from the pipeline’s settings page:
+
+<%= image "github-settings.png", width: 1096/2, height: 1820/2, alt: "Screenshot of the Buildkite GitHub settings" %>
+
+If you need to set up the webhook again, you can find the instructions linked at the bottom of the pipeline GitHub settings page.
+
+## Running builds on Pull Requests
+
+To run builds for GitHub Pull Requests, edit the the GitHub settings for your Buildkite pipeline:
+
+- Choose the 'build pull requests' checkbox.
+- Optionally, limit Pull Request builds to certain branches.
+- Optionally, skip Pull Request Builds for committs that have already been built.
+- Optionally, build pull requests that are ready for review.
+- Optionally, build pull requests for forked repositories. Make sure to take a look at the [managing pipeline secrets](/docs/pipelines/secrets) guide if you choose to do this.
```

Comment:
Could we do the above as like this one is required, then here's a list of the optional ones?

Reply:
I'm not quite sure what you mean here. Do you have an example? 

I can drag out the Optionally from each item and say here is a list of optional configs?

Reply:
Yup, that's what I was getting at 😅